test/unit/browser/vscode.test.ts

```diff
@@ -58,4 +62,117 @@ describe("vscode", () => {
       document.body.removeChild(mockElement)
     })
   })
+  describe("setBodyBackgroundToThemeBackgroundColor", () => {
+    beforeEach(() => {
+      // We need to set the url in the JSDOM constructor
+      // to prevent this error "SecurityError: localStorage is not available for opaque origins"
+      // See: https://github.com/jsdom/jsdom/issues/2304#issuecomment-622314949
+      const { window } = new JSDOM("", { url: "http://localhost" })
```

Comment:
I guess this is because Node.js doesn't have `window`?

Reply:
Exactly! `window` is only available in browser environments because it's a browser global